langchain4j-core/src/test/java/dev/langchain4j/model/chat/common/AbstractBaseChatModelIT.java

```diff
@@ -1294,6 +1295,58 @@ protected void should_respect_JSON_response_format_with_schema(M model) {
         }
     }
 
+    @ParameterizedTest
+    @MethodSource("models")
+    @EnabledIf("supportsJsonResponseFormatWithNativeSchema")
+    protected void should_respect_JsonNativeSchema_responseFormat(M model) throws Exception {
+        var rawSchema =
+                """
+            {
+                "$schema": "http://json-schema.org/draft-07/schema#",
+                "type": "object",
+                "properties": {
+                    "city": {
+                        "type": "string"
+                    }
+                },
+                "required": ["city"],
+                "additionalProperties": false
+            }""";
+
+        ResponseFormat schemaFormat = ResponseFormat.builder()
+                .type(ResponseFormatType.JSON)
+                .jsonSchema(JsonSchema.builder()
+                        .name("Answer")
+                        .rootElement(
+                                JsonNativeSchema.builder().schema(rawSchema).build())
```

Comment:
```suggestion
                                JsonNativeSchema.from(rawSchema))
```
Would be nice to add this helper method